client/src/components/ContentNavigator/ContentModel.ts

```diff
@@ -8,6 +8,7 @@ import axios, {
   AxiosRequestConfig,
   AxiosResponse,
 } from "axios";
+import mime from "mime";
```

Comment:
This is giving me issues when running tests:

> > Error [ERR_REQUIRE_ESM]: require() of ES Module /home/runner/work/vscode-sas-extension/vscode-sas-extension/client/node_modules/mime/dist/src/index.js from /home/runner/work/vscode-sas-extension/vscode-sas-extension/client/out/src/components/ContentNavigator/ContentModel.js not supported.

@smorrisj @scnwwu I know this has come up before. I can't recall if we had a solution/what it might be? 

As an alternative, we can always just use a subset of this list, or the list in its entirety - https://github.com/micnic/mime.json/blob/master/index.json

Reply:
it seems that vscode doesn't allow the consumption of esm modules, but that's on its way to being resolved - https://github.com/microsoft/vscode/issues/130367. 

Since mime types are pretty static, I just grabbed the json file above and used that (note: mime.json isn't in the npm registry) in https://github.com/sassoftware/vscode-sas-extension/pull/914/commits/03229bd86e4f471e8f5cf7f4da5b882a34e414ef